### ISSUE TITLE
Getting a NullPointer with Fallback protocol

### ISSUE THREAD

**Issue by abbasvalliani** [chhsiao90/nitmproxy#55]
```
14:51:35.968 [nioEventLoopGroup-3-3] DEBUG c.g.c.n.h.p.tls.TlsFrontendHandler - SSL detection with true
14:51:35.995 [nioEventLoopGroup-3-3] DEBUG c.g.c.n.h.p.tls.TlsFrontendHandler - Client SNI lookup with *****.dev
14:51:35.995 [nioEventLoopGroup-3-2] DEBUG c.g.c.n.h.p.h.Http1FrontendHandler - [Client (kubernetes.docker.internal:61477)] <=> [Server (*****.dev:443)] : HttpObjectAggregator$AggregatedFullHttpRequest(decodeResult: success, version: HTTP/1.1, content: CompositeByteBuf(ridx: 0, widx: 0, cap: 0, components=0))
GET /static/js/main.chunk.js HTTP/1.1
Host: *****.dev
Connection: keep-alive
sec-ch-ua: " Not A;Brand";v="99", "Chromium";v="90", "Google Chrome";v="90"
If-None-Match: W/"12de5b-LJNNHxiMdOHBcmHz0IYO2ViROjI"
sec-ch-ua-mobile: ?0
User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.212 Safari/537.36
Accept: */*
Sec-Fetch-Site: same-origin
Sec-Fetch-Mode: no-cors
Sec-Fetch-Dest: script
Referer: https://*****.dev/dashboard
Accept-Encoding: gzip, deflate, br
Accept-Language: en-US,en;q=0.9
Cookie: __stripe_mid=9d05fbef-3240-46ab-8fb1-044b78ecb9b3b9a2c6; __stripe_sid=b361d74a-688f-4729-aeb0-fd9fb55f59d47e5852
content-length: 0
14:51:35.997 [nioEventLoopGroup-3-3] DEBUG c.g.c.n.h.p.tls.TlsBackendHandler - [Client (kubernetes.docker.internal:61482)] <=> [Server (*****.dev:443)] : handlerAdded
14:51:36.013 [nioEventLoopGroup-3-3] DEBUG c.g.c.n.h.p.tls.TlsFrontendHandler - Client ALPN lookup with [h2, http/1.1]
14:51:36.022 [nioEventLoopGroup-3-1] INFO  c.g.c.n.listener.HttpEventLogger - 80 *****.dev 304 Not Modified 0 GET /static/js/0.chunk.js HTTP/1.1 *****.dev
14:51:36.025 [nioEventLoopGroup-3-2] INFO  c.g.c.n.listener.HttpEventLogger - 30 *****.dev 304 Not Modified 0 GET /static/js/main.chunk.js HTTP/1.1 *****.dev
14:51:36.062 [nioEventLoopGroup-3-3] DEBUG c.g.c.n.h.p.tls.TlsFrontendHandler - ALPN negotiated with http/1.1
14:51:36.125 [nioEventLoopGroup-3-3] DEBUG c.g.c.n.h.p.h.Http1BackendHandler - [Client (kubernetes.docker.internal:61482)] <=> [Server (*****.dev:443)] : handlerAdded
14:51:36.128 [nioEventLoopGroup-3-3] DEBUG c.g.c.n.h.p.tls.TlsBackendHandler - [Client (kubernetes.docker.internal:61482)] <=> [Server (*****.dev:443)] : handlerRemoved
14:51:36.143 [nioEventLoopGroup-3-3] DEBUG c.g.c.n.h.p.h.Http1FrontendHandler - [Client (kubernetes.docker.internal:61482)] <=> [Server (*****.dev:443)] : handlerAdded
14:51:36.144 [nioEventLoopGroup-3-3] DEBUG c.g.c.n.h.p.tls.TlsFrontendHandler - [Client (kubernetes.docker.internal:61482)] <=> [Server (*****.dev:443)] : handlerRemoved
14:51:36.466 [nioEventLoopGroup-3-4] DEBUG c.g.c.n.h.p.tls.TlsFrontendHandler - [Client (kubernetes.docker.internal:61486)] <=> [PROXY] : handlerAdded
14:51:36.471 [nioEventLoopGroup-3-4] DEBUG c.g.c.n.h.p.tls.TlsFrontendHandler - SSL detection with true
14:51:36.471 [nioEventLoopGroup-3-4] DEBUG c.g.c.n.h.p.tls.TlsFrontendHandler - Client SNI lookup with *****.dev
14:51:36.473 [nioEventLoopGroup-3-4] DEBUG c.g.c.n.h.p.tls.TlsBackendHandler - [Client (kubernetes.docker.internal:61486)] <=> [Server (*****.dev:443)] : handlerAdded
14:51:36.474 [nioEventLoopGroup-3-4] DEBUG c.g.c.n.h.p.tls.TlsFrontendHandler - Client ALPN lookup with null
14:51:36.480 [nioEventLoopGroup-3-4] WARN  i.n.util.concurrent.DefaultPromise - An exception was thrown by com.github.chhsiao90.nitmproxy.handler.protocol.tls.TlsBackendHandler$$Lambda$1398/0x0000000800d9ec40.operationComplete()
java.lang.NullPointerException: null
        at com.github.chhsiao90.nitmproxy.handler.protocol.tls.TlsBackendHandler.getFallbackProtocol(TlsBackendHandler.java:145)
        at com.github.chhsiao90.nitmproxy.handler.protocol.tls.TlsBackendHandler.configSsl(TlsBackendHandler.java:137)
        at com.github.chhsiao90.nitmproxy.handler.protocol.tls.TlsBackendHandler.lambda$handlerAdded$1(TlsBackendHandler.java:48)
        at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
        at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
        at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
        at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
        at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
        at io.netty.util.concurrent.DefaultPromise.setSuccess(DefaultPromise.java:96)
        at com.github.chhsiao90.nitmproxy.handler.protocol.tls.TlsFrontendHandler$AlpnNegotiateHandler.lookup(TlsFrontendHandler.java:205)
        at com.github.chhsiao90.nitmproxy.handler.protocol.tls.AbstractAlpnHandler.lookup(AbstractAlpnHandler.java:109)
        at io.netty.handler.ssl.SslClientHelloHandler.select(SslClientHelloHandler.java:188)
        at io.netty.handler.ssl.SslClientHelloHandler.decode(SslClientHelloHandler.java:149)
        at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:508)
        at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:447)
        at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:276)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
        at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
        at com.github.chhsiao90.nitmproxy.handler.protocol.tls.TlsFrontendHandler$ConnectToRemote.channelRead(TlsFrontendHandler.java:168)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
        at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
        at io.netty.handler.codec.ByteToMessageDecoder.handlerRemoved(ByteToMessageDecoder.java:253)
        at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:515)
        at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:447)
        at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:276)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
        at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
        at io.netty.handler.codec.ByteToMessageDecoder.handlerRemoved(ByteToMessageDecoder.java:253)
        at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:515)
        at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:447)
        at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:276)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
        at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
        at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
        at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919)
        at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:166)
        at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:719)
        at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:655)
        at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:581)
        at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:493)
        at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989)
        at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
        at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
        at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by abbasvalliani** [comment]
Please let me know if I can provide more details. Thanks again for all your help.

**Comment by chhsiao90** [comment]
Thanks for reporting the issue, bug fixed and released in 1.3.2.

**Comment by abbasvalliani** [comment]
Thanks!
